syncstorage/tests/functional/test_storage.py

```diff
@@ -142,6 +142,9 @@ def test_get_collection(self):
         res = self.app.put_json(self.root + '/storage/col2/128', bso)
         ts1 = float(res.headers["X-Last-Modified"])
 
+        # delay to ensure unique timestamp
+        time.sleep(0.01)
+
```

Comment:
So in this case, I believe that a server generating the same timestamp for these requests is in violation of the protocol spec.  We promise the client that sequential write requests will see a monotonically-increasing last-modified timestamp:

  http://docs.services.mozilla.com/storage/apis-1.5.html#concurrency-and-conflict-management

(Yeah, this is non-trivial from the server's point of view, but it's even *more* non-trivial to ask the client to consistently do the right thing here)